src/Extensions/LICENSE.TXT

```diff
@@ -0,0 +1,23 @@
+The MIT License (MIT)
```

Comment:
Do not create a separate license file here. This is still part of the dotnet-monitor repo so it should be using the files at the root.